applications/nrf5340_audio/src/bluetooth/bt_management/bt_mgmt.c

```diff
@@ -295,10 +297,12 @@ static int random_static_addr_cfg(void)
 	}
 
 	LOG_WRN("Unable to read from FICR");
+
 	/* If no address can be created based on FICR,
-	 * then a random address is created
+	 * then a random address can be created
```

Comment:
I think 'is' is better as it **will** be created

Reply:
Did some changes to this comment